nservicebus/azure-storage-persistence/index.md

```diff
@@ -35,4 +35,19 @@ To ensure that only one saga can be created for one correlation property value,
 
 If a migration from 6.2.3 or earlier to a newer version was performed without applying [saga deduplication](/nservicebus/upgrades/asp-saga-deduplication.md), `DuplicatedSagaFoundException` can be observed because of duplicates violating a unique property of a saga. The exception message will include all the information to track down the error for example: `Sagas of type MySaaga with the following identifiers 'GUID1', 'GUID2' are considered duplicates because of the violation of the Unique property CorrelationId.`.
 
-The way to address them is go through the upgrade guide linked above.
\ No newline at end of file
+The way to address them is go through the upgrade guide linked above.
+
+### Supported saga properties' types
+
+Azure Storage Persistence supports exactly the same set of types as [Azure Table Storage](https://msdn.microsoft.com/library/azure/dd179338.aspx). The supported types are following:
+
+ 1. `byte[]`
+ 1. `bool`
+ 1. `DateTime`
+ 1. `Guid`
+ 1. `int`
+ 1. `long`
+ 1. `double`
+ 1. `string`
+
+When a saga containing a property of an unsupported type is persisted, an exception containing a following information is thrown: `The property type 'the_property_name' is not supported in windows azure table storage`. If an object of a complex type is required to be stored, then it's a user responsibility to serialize/deserialize the value.
```

Comment:
What about users receiving this exception? Wouldn't it be helpful to be directed to our docs? 
Additionally, we'd like to provide at least some guidance how it can be overcome.